src/locales/en/pages.yml

```diff
@@ -6,3 +6,6 @@ could-not-match-national-insurance:
   content:
     - We could not find a match for your National Insurance number when we checked with HMRC.
     - You can check your National Insurance number and try again. You’ll only be able to do this once.
+abandon:
+  content: " "
+  h1: "What would you like to do?"
```

Comment:
The page title seems to be an untranslated string, are we maybe missing a content key? Or is that going to be handled elsewhere?

![image](https://github.com/govuk-one-login/ipv-cri-check-hmrc-front/assets/40401118/8e2904a8-51ee-4c17-9768-79f576bc605b)

Reply:
updated as per last commit